.github/workflows/flutter-build.yml

```diff
@@ -1421,28 +1415,44 @@ jobs:
             fi
             popd
             popd
-            $VCPKG_ROOT/vcpkg install --triplet ${{ matrix.job.vcpkg-triplet }} --x-install-root="$VCPKG_ROOT/installed"
             # rust
             pushd /opt
             # do not use rustup, because memory overflow in qemu
-            wget -O rust.tar.gz https://static.rust-lang.org/dist/rust-${{env.RUST_TOOLCHAIN_VERSION}}-${{ matrix.job.target }}.tar.gz
+            wget --output-document rust.tar.gz https://static.rust-lang.org/dist/rust-${{env.RUST_TOOLCHAIN_VERSION}}-${{ matrix.job.target }}.tar.gz
             tar -zxvf rust.tar.gz > /dev/null && rm rust.tar.gz
-            cd rust-${{env.RUST_TOOLCHAIN_VERSION}}-${{ matrix.job.target }} && ./install.sh
+            pushd rust-${{env.RUST_TOOLCHAIN_VERSION}}-${{ matrix.job.target }}
+            ./install.sh
+            popd
             rm -rf rust-${{env.RUST_TOOLCHAIN_VERSION}}-${{ matrix.job.target }}
-            # edit config
+            popd
```

Comment:
popd again?

Reply:
Yes, because the original line was 'cd rust-… && ./install.sh'and I splitted it to `pushd rust-… ` and `./install.sh` so one more popd is needed to go back to /workspace or /

Reply:
There is very simple method to check we land in the same dir we started: count that starting from `run:` statement, count of pushd == count of popd

Reply:
it seems break x64

Reply:
It breaks both because suddenly /opt/artifacts is not generated outside… Should be easy to fix, but I assume reeulting debs are stored in workspace dir?